kotlin/android/app/src/main/java/dev/firezone/android/features/session/ui/ViewResource.kt

```diff
@@ -0,0 +1,29 @@
+package dev.firezone.android.features.session.ui
+
+import dev.firezone.android.tunnel.model.Resource
+import dev.firezone.android.tunnel.model.Site
+import dev.firezone.android.tunnel.model.StatusEnum
+
+data class ViewResource(
+    val id: String,
+    val address: String,
+    val addressDescription: String?,
+    val sites: List<Site>?,
+    val name: String,
+    val status: StatusEnum,
+    var enabled: Boolean = true,
+    var disableable: Boolean = true,
+    )
+
+fun Resource.toViewResource(): ViewResource {
```

Comment:
My reasoning was:

* I think in kotlin `toX` is more common than `fromX` because there are no static methods, though I think something like that can be achieved with `companion` classes? But `toViewResources` seemed more idiomatic
* I think it's clearer what is a `Resource` from a `ViewResource` module rather than what's a `ViewResource` from a `Resource` module.